src/sentry/tagstore/current/__init__.py

```diff
@@ -0,0 +1,12 @@
+"""
+sentry.tagstore.current
```

Comment:
I don't like the backend name `current` but I have no other ideas for "another backend that uses Postgres but differently than `legacy`"

Reply:
v2 🥈